<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addGoogleBookDialog</class>
 <widget class="QDialog" name="addGoogleBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>871</width>
     <height>731</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; text-decoration: underline;&quot;&gt;Recherche&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="titleSearchLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Titre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleSearchLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchPushButton">
        <property name="text">
         <string>Recherche</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="authorSearchLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Auteur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="authorSearchLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="nbrFindBookLabel">
        <property name="text">
         <string>x livre(s) trouvé(s):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="nbQuerylcdNumber">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; text-decoration: underline;&quot;&gt;Informations &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="resultLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="titreLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="titleCheckBox">
          <property name="text">
           <string>Titre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="titleLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="categoriesLayout_2">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="agLayout_2">
          <item>
           <widget class="QCheckBox" name="authorsCheckBox">
            <property name="text">
             <string>Auteur(s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="authorLineEdit"/>
          </item>
          <item>
           <widget class="QCheckBox" name="genderCheckBox">
            <property name="text">
             <string>Genre</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="genderLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ecLayout_2">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="editorCheckBox">
            <property name="text">
             <string>Editeur</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editorLineEdit"/>
          </item>
          <item>
           <widget class="QCheckBox" name="collectionCheckBox">
            <property name="text">
             <string>Collection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="collectionLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="summaryCheckBox">
        <property name="text">
         <string>Résumé</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="summaryPlainTextEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="neaptcLayout_2">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="neapLayout_2">
          <item>
           <layout class="QHBoxLayout" name="tomesLayout">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="tomeLayout">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="tomeCheckBox">
                <property name="text">
                 <string>Tome</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="tomeLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="tomeEndLayout">
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Sur</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="tomeEndLineEdit"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="yearCheckBox">
            <property name="text">
             <string>Année de parution</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="yearLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="tomecodeLayout_2">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="isbnCheckBox">
            <property name="text">
             <string>Code ISBN-13</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="isbnLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="imageLayout">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QCheckBox" name="imgCheckBox">
          <property name="text">
           <string>Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        <property name="centerButtons">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
